src/main/java/net/sf/jabref/logic/openoffice/OOBibStyle.java

```diff
@@ -149,12 +149,11 @@ public OOBibStyle(File styleFile, LayoutFormatterPreferences prefs,
         path = styleFile.getPath();
     }
 
-    public OOBibStyle(String resourcePath, LayoutFormatterPreferences prefs)
-            throws IOException {
+    public OOBibStyle(String resourcePath, LayoutFormatterPreferences prefs) throws IOException {
         this.prefs = Objects.requireNonNull(prefs);
         this.encoding = StandardCharsets.UTF_8;
         setDefaultProperties();
-        initialize(OOBibStyle.class.getResource(resourcePath).openStream());
+        initialize(OOBibStyle.class.getResourceAsStream(resourcePath));
```

Comment:
Maybe we can wrap it inside `Objects.requireNonNull()` and also add some `Objects.requireNonNull` for `resourcePath` and `prefs` before lin 153